### ISSUE TITLE
About your model structure in paper

### ISSUE THREAD

**Issue by Ivan-VV** [LanglandsLin/MS2L#3]
Thanks for your great work! 
Why is the model structure in the code different from that in your paper? In the paper, the shared encoder is a 1-layer bidirectional GRU with 30 units in each layer, but it is a 2-layer GRU with 600 units in the code. The classifier in your paper includes a 1-layer unidirectional GRU with 60 units and a MLP for recognition, but it is just a 2-layer FC in your code. The multi-task heads in your code are also different from those in your paper. 
So why are they different? Which model structure  is used to get the performance in your paper? Thanks the help!

**Comment by LanglandsLin** [comment]
Hi @Ivan-VV!

Thank you for your attention! 

This is an error in the paper. We adjust the network structure in the follow-up research. So what is actually used is the structure in the code. Because of the following:

1. The 30-unit BiGRU cannot handle large data sets.

2. The linear evaluation of the unsupervised learning of the previous works applies a layer of FC for classification after the encoder is fixed. Therefore, we also change the classifier to a layer of FC.

3. The reconstruction head is replaced with GRU. Because we find that a stronger reconstruction head can get better performance.

Sorry for the confusion. Looking forward to more discussions.

**Comment by Ivan-VV** [comment]
Thank you very much for your reply！ @LanglandsLin 
I also have some questions about the details of training. Could you please tell me the learning rate and epoch-num under the pre-training and fine-tuning settings. I didn't find them in your paper. 
In the code, they are 1e-3 and 60 respectively, but I don't know whether they are suitable for both pre-training and fine-tuning (including moving pretraining strategy and jointly training strategy).
Looking forward to your reply!

**Comment by LanglandsLin** [comment]
We apply 60 epochs for pre-training and 300 epochs for linear evaluation or fine-tuning.

**Comment by Ivan-VV** [comment]
Thank you for your reply! @LanglandsLin
